### PR TITLE
Delete lit dependency on range-slider

### LINKED ISSUE #939: Rewrite range-slider to not use lit

### PR DESCRIPTION
**Pull request by fatih-erikli**
Fixes #939.

### CODE REVIEW

**Comment by justvanrossum** [comment]
To avoid unnecessary rerendering, should we perhaps implement a setter/getter for value, instead of doing the automatic render via the properties list?

**Comment by fatih-erikli** [comment]
Ok. I will try that.

**Comment by fatih-erikli** [comment]
To avoid such rerenders, we may consider using a simple dom diff-patch library in future. This looks like a clean implementation of what react, vue do. https://github.com/tjcchen/simple-virtual-dom/blob/main/src/main.js#L49-L52

**Comment by fatih-erikli** [comment]
I don't know how lit solves this problem.

File: src/fontra/client/web-components/range-slider.js [hunk 1]
```diff
@@ -162,169 +160,172 @@ export class RangeSlider extends LitElement {
...
-          />
-        </div>
-      </section>
-    `;
+    return round(this.value, decimalPlaces);
+  }
+
+  set value(value) {
+    this._value = value;
+    if (this.rangeInput) this.rangeInput.value = value;
```
Comment (justvanrossum) [inline src/fontra/client/web-components/range-slider.js:177]: Please no one-line if statements: use a proper { } block. Same for the line below.

(We need to check if there's a prettier option for that)

**Pull request merged**